app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -13,5 +13,6 @@
     //Default maximum time limit for editing message(Same as server)
     public final static int DEFAULT_MAXIMUM_CONTENT_EDIT_LIMIT = 600;
     public final static boolean DEFAULT_EDITING_ALLOWED = true;
+    public static String serverUrl = null;
```

Comment:
Something like this

`public final static String SERVER_URL = "SERVER_URL";`

and use this ` i.putExtra(SERVER_URL, app.getServerURI());` and
`String serverUrl = getIntent().getStringExtra(SERVER_URL);`